examples/demo_vectors_coord-like.py

```diff
@@ -0,0 +1,41 @@
+"""
```

Comment:
Can you rename this example `add_vectors_coords` and as a minor edit make the text at the top look a little more like the text at the top of that `add_images` and `add_markers` examples (i.e. replace `This example ... ` with `Display ...`)